test/handler.test.js

```diff
@@ -594,6 +650,35 @@ describe('strong-error-handler', function() {
         });
     });
 
+    it('sanitizes response to prevent XSS', function(done) {
```

Comment:
Please make this test name more descriptive - e.g. `HTML-escapes all 4xx response properties in production mode`.